light/client_test.go

```diff
@@ -1044,7 +1065,7 @@ func TestClientEnsureValidHeadersAndValSets(t *testing.T) {
 		)
 		require.NoError(t, err)
 
-		_, err = c.VerifyLightBlockAtHeight(3, bTime.Add(2*time.Hour))
+		_, err = c.VerifyLightBlockAtHeight(context.Background(), 3, bTime.Add(2*time.Hour))
```

Comment:
It's a lot less typing to just define a variable `ctx` and use that in all the calls. Not something that needs to be changed right now, just a suggestion for later.